hashicorp/dev-portal
# feat: stage vagrant vmware page

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksvagrant-vmware-page-hashicorp.vercel.app/vagrant/downloads/vmware) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1202888288953409) 🎟️
- [Figma](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=13122%3A171549) 🎨 

## 🗒️ What

Adds the vagrant vmware install page. 

## 🛠️ How

Adjusted the current install page functions to account for the various edge cases. 

## 📸 Design Screenshots
<img width="774" alt="Screen Shot 2022-09-12 at 11 38 58 AM" src="https://user-images.githubusercontent.com/36613477/189730914-b55c35ac-af0e-4ce1-8021-acf511d0487d.png">


## 🧪 Testing

- [Visit this page ](https://dev-portal-git-ksvagrant-vmware-page-hashicorp.vercel.app/vagrant/downloads/vmware)
- Ensure the download links work as expected
- Also visit the[ install page](https://dev-portal-git-ksvagrant-vmware-page-hashicorp.vercel.app/vagrant/downloads)
- Check that the vmware merchandising slot is rendering as expected.

## 💭 Anything else?


~~I'm not sure what we want this to say? Should we remove it or just leave it to the content folks. Looks like it's auto-generated.~~ I removed this.
<img width="928" alt="Screen Shot 2022-09-12 at 11 37 31 AM" src="https://user-images.githubusercontent.com/36613477/189730686-941d108c-0c26-4884-978a-e2b530b60bb2.png">
